### ISSUE TITLE
AvFeedback is not shown for AvSelect

### ISSUE THREAD

**Issue by pashazz** [Availity/availity-react#24]
Hello. I have a regular Input (type select) that is defined like this:

```jsx
 <AvGroup>
  <InputGroup>
    <InputGroupAddon style={ {"line-height": "1!important"}} addonType="prepend">
      <InputGroupText style = { { height: '100%'}}>
        <FontAwesomeIcon icon={faServer}/>
      </InputGroupText>
    </InputGroupAddon>
    <AvInput type="select" id="pool-select" name="pool-select" onChange={onChange} required>
      <option value="">Select where to deploy instance</option>
      {
        pools.map(pool =>
          <option key={pool.uuid} value={pool.uuid}>{pool.description}</option>)
      }
    </AvInput>
    <AvFeedback>Select a pool to install in</AvFeedback>
  </InputGroup>
  </AvGroup>;

```

If I leave it empty, the form won't submit and feedback is shown:
![selection_008](https://user-images.githubusercontent.com/45922/44212365-d31cef80-a173-11e8-9bdd-90cebff24336.png)

I created AvSelect component the same way:

```jsx
     <AvGroup>
    <InputGroup style={ {padding: '10px'}}>
      <InputGroupAddon addonType="prepend">
        <InputGroupText>
          <FontAwesomeIcon icon={faDesktop}/>
        </InputGroupText>
      </InputGroupAddon>
    <SelectList
      style={{flex: 1}}
      data={templates}
      onChange={onChange}
      placeholder="Select Template..."
      name="template"
      id="template"
      required
    />
      <AvFeedback>Select a template to use as configuration basis</AvFeedback>
    </InputGroup>

     </AvGroup>
```

SelectList is a component that injects options for AvSelect:
```jsx
   <div style={{flex: 1}}>
        <AvSelect options={this.state.options}
                {...props} />
      </div>
```

But If I leave it empty,the form won't send with  the AvFeedback text  not shown,  all selectable options are painted red instead:

![selection_009](https://user-images.githubusercontent.com/45922/44212865-29d6f900-a175-11e8-94b3-2277f08d34ff.png)

Packages:

```
cat package-lock.json| grep avail                                    
    "@availity/reactstrap-validation-select": {
      "resolved": "https://registry.npmjs.org/@availity/reactstrap-validation-select/-/reactstrap-validation-select-1.5.1.tgz",
    "availity-reactstrap-validation": {
      "resolved": "https://registry.npmjs.org/availity-reactstrap-validation/-/availity-reactstrap-validation-2.0.6.tgz",
```

**Comment by GoPro16** [comment]
Sorry for the late reply! 😄 

The reason this is happening is because `@availity/reactstrap-validation-select` doesn't use the native `input` field which applies the `form-control` className onto the element. This is how feedback gets rendered to the DOM.

The way that we made this work for `AvSelectField` is we check for the `errorMessage` and manually add the `d-block` classname on to the `AvFeedback` component.

https://github.com/Availity/availity-react/blob/master/packages/reactstrap-validation-select/AvSelectField.js#L47-L48

Check out the below code sandbox if you are wanting to use your own custom `AvSelect` with an input group.
https://codesandbox.io/s/availity-react-issue-24-q20md


## TLDR

We should make it easier for having a custom input group with `AvSelect`.
